src/aliceVision/image/io.cpp

```diff
@@ -213,7 +213,6 @@ void readImage(const std::string& path,
 
   if(imageColorSpace == EImageColorSpace::SRGB) // color conversion to sRGB
   {
-      ALICEVISION_LOG_INFO("Convert image " << path << " from " << colorSpace << " to sRGB colorspace");
     if (colorSpace != "sRGB")
     {
       oiio::ImageBufAlgo::colorconvert(inBuf, inBuf, colorSpace, "sRGB");
```

Comment:
BTW, it could be better to also change the other ones from `ALICEVISION_LOG_INFO` to `ALICEVISION_LOG_TRACE`?